app/errors.js

```diff
@@ -54,3 +54,15 @@ exports.isNotAdmin = {
   statusCode: 422,
   message: 'The user is not an user administrator.'
 };
+
+exports.photosOfAlbumsProviderFail = {
+  statusCode: 422,
```

Comment:
I would put a 5xx error. And this is has the same meaning of the error 'albumsProviderFail' from above, maybe we should keep only one error.